include/production/ProductionValues.hpp

```diff
@@ -0,0 +1,20 @@
+#ifndef SHARED_VALUES_HPP
+#define SHARED_VALUES_HPP
+
+#include <cppQueue.h>
+
+#include "DolibarrClient.hpp"
+
+struct ProductionValues {
+  portMUX_TYPE subTaskLock;
+  // RFID reader values
+  cppQueue tags;
+  // API client values
+  DolibarrClientStatus dolibarrClientStatus;
+  int targetWarehouse;
+
+  ProductionValues() : tags(sizeof(int), 3, cppQueueType::FIFO), targetWarehouse(3), subTaskLock(portMUX_INITIALIZER_UNLOCKED) {}
```

Comment:
You should put initialization of the fields in the declaration order (subTaskLock -> tags -> dolibarrClientStatus -> targetWarehouse).
Also, you forgot to initialize dolibarrClientStatus.